slemicro/create_vm.sh

```diff
@@ -3,19 +3,40 @@ set -euo pipefail
 BASEDIR="$(dirname "$0")"
 
 die(){
-	echo ${1}
+	echo ${1} 1>&2
 	exit ${2}
 }
 
+usage(){
+	cat <<-EOF
+	Usage: ${0} [-f <path/to/variables/file>] [-n <vmname>]
+	EOF
+}
+
+while getopts 'f:n:h' OPTION; do
+	case "${OPTION}" in
+		f)
+			[ -f "${OPTARG}" ] && ENVFILE="${OPTARG}" || die "Parameters file ${OPTARG} not found" 2
+			;;
+		n)
+			VMNAME="${OPTARG}"
+			;;
+		h)
+			usage && exit 0
+			;;
+		?)
+			usage && die "Parameter ${OPTION} wrong" 3
```

Comment:
The current version would print something like:

```
./create_vm.sh: illegal option -- s
Usage: ./create_vm.sh [-f <path/to/variables/file>] [-n <vmname>]
Parameter ? wrong
```
This way the input parameter is not properly handled in our custom message and we also have the default `illegal option` output which should be sufficient.

This makes the call to `die()` rather unnecessary so replacing it with a standard `exit` might be better (`exit 2` indicating invalid usage of the command). WDYT?


```suggestion
			usage && exit 2
```